SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/TestActors/ReplicatedTestPawnBase_RepGraphAlwaysReplicate.cpp

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "ReplicatedTestPawnBase_RepGraphAlwaysReplicate.h"
+
+AReplicatedTestPawnBase_RepGraphAlwaysReplicate::AReplicatedTestPawnBase_RepGraphAlwaysReplicate()
+{
+	bReplicates = true;
```

Comment:
ADefaultPawn already replicates by default, so we can remove this constructor and cpp file fully